backend/editor/scheduler.py

```diff
@@ -0,0 +1,35 @@
+import contextlib
+import logging
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
+from .controllers.project_controller import delete_project, get_projects_by_status
+from .github_functions import GithubOperations
+from .graph_db import TransactionCtx
+from .models.project_models import ProjectStatus
+
+log = logging.getLogger(__name__)
+
+
+async def delete_merged_projects():
+    async with TransactionCtx():
+        exported_projects = await get_projects_by_status(ProjectStatus.EXPORTED)
+        for project in exported_projects:
+            pr_number = project.github_pr_url and project.github_pr_url.split("/")[-1]
+            if not pr_number:
+                log.warning(f"PR number not found for project {project.id}")
+                continue
+            github_object = GithubOperations(project.taxonomy_name, project.branch_name)
+            if await github_object.is_pr_merged(int(pr_number)):
+                await delete_project(project.id)
```

Comment:
As those methods can fail, I would add a try / except with log error, to avoid having one project blocking all other projects removal.

Reply:
I don't think the overall function should fail (unless there is a connection problem to Github) but I grouped the tasks into `asyncio.gather` to run them independantly